modules/ixBidAdapter.js

```diff
@@ -285,24 +285,26 @@ function getBidRequest(id, impressions) {
  * From the userIdAsEids array, filter for the ones our adserver can use, and modify them
  * for our purposes, e.g. add rtiPartner
  * @param {array} allEids userIdAsEids passed in by prebid
+ * @param {object} flocId flocId passed in by prebid
  * @return {object} contains toSend (eids to send to the adserver) and seenSources (used to filter
  *                  identity info from IX Library)
  */
-function getEidInfo(allEids) {
+function getEidInfo(allEids, flocData) {
   // determines which eids we send and the rtiPartner field in ext
-  var sourceRTIMapping = {
+  const sourceRTIMapping = {
```

Comment:
This and the `PROVIDERS` variable in `_getUserIds` could still be a file-level const if we wanted to avoid re-creating on every call.